client/components/popover/index.jsx

```diff
@@ -361,7 +361,7 @@ class Popover extends Component {
 	}
 
 	setPopoverId( id ) {
-		this.id = id || `pop__${ uid( 16 ) }`;
+		this.id = id || `pop__${ uuid() }`;
```

Comment:
This line is almost unneeded. If we think that it's better to get rid of it let's do it. I could do it in a next PR as well.

Reply:
Yeah, `lodash/uniqueId` seems better (and simpler) for this case. It actually guarantees that it will be unique, while `uuid` can't.